sci-misc/boinc/metadata.xml

```diff
@@ -19,7 +19,8 @@
   Use nvidia cuda toolkit for speeding up computations.
   NOTE: works only for subset of nvidia graphic cards so make sure your card
   is supported before opening a bug about it.
-  </flag>
+    </flag>
+    <flag name="opencl">Enable OpenCL support</flag>
```

Comment:
I find this description false since it doesn't actually enable or disable anything, just changes dependencies. I should also point out that using USE flags to adjust runtime dependencies is strongly discouraged, and practically banned for anything that gets built from source.